localstack/aws/handlers/routes.py

```diff
@@ -29,4 +29,4 @@ def __call__(self, chain: HandlerChain, context: RequestContext, response: Respo
             chain.stop()
         except NotFound:
             if self.respond_not_found:
-                chain.respond(404)
+                chain.respond(404, "not found")
```

Comment:
actually 404 requests for WebsocketRequests need to be dealt with separately, since the default implementation of `ROUTER` actually has `respond_not_found=False`, and falls back to the EmptyResponseHandler.